tests/remeshing/parallel/implicit/acceleration/helper.hpp

```diff
@@ -16,17 +16,18 @@ using testing::operator""_write;
 
 namespace acceleration {
 
-inline void runResetA(testing::TestContext &context)
+// The data written in the acceleration tests are always 40, 30, 15
+// first and second are read data from the first and second iteration
+// The end of a timewindow and the end of the simulation are never accelerated and thus 15
```

Comment:
@uekerman what are your thoughts on this testing method?

Reply:
Should have a certain test coverage already, yes. In a single time window IQN-ILS and IQN-IMVJ are identical (as you also observed). To really test IQN-IMVJ, you could only remesh every second time window.

Reply:
Added a second time window before and after the reinit.
Despite the same data, the IQNs result in different data as the initial data is once 0 and once the result of the 2nd time window.